nixos/modules/services/hardware/amdvlk.nix

```diff
@@ -0,0 +1,61 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.hardware.amdgpu.amdvlk;
+in {
+  options.hardware.amdgpu.amdvlk = {
+    enable = lib.mkEnableOption "AMDVLK Vulkan driver";
+
+    package = lib.mkPackageOption pkgs "amdvlk" { };
+
+    supportExperimental.enable = lib.mkEnableOption "Experimental features support";
+
+    support32Bit.enable = lib.mkEnableOption "32-bit driver support";
+    support32Bit.package = lib.mkPackageOption pkgs.driversi686Linux "amdvlk" { };
```

Comment:
to fix 
![image](https://github.com/NixOS/nixpkgs/assets/140964/18ffcd02-f263-4413-95f1-fec81ffbd069)


```suggestion
    support32Bit.package = lib.mkPackageOption pkgs [ "driversi686Linux" "amdvlk" ] { };
```

Reply:
Fixed in https://github.com/NixOS/nixpkgs/pull/318175/commits/56123e0d5b87a743df1aa82e5ad462cfbaaaa1b7